packages/manager/modules/vps/src/backup-storage/add/add-backup-storage.html

```diff
@@ -5,63 +5,40 @@
         data-secondary-action="$ctrl.cancel()"
         data-secondary-label="{{ ::'vps_common_cancel' | translate }}"
         data-on-dismiss="$ctrl.cancel()"
-        data-loading="$ctrl.ipBlocks.loading || $ctrl.addStorage.loading"
-    >
+        data-loading="$ctrl.ipBlocks.loading || $ctrl.addStorage.loading">
+
         <div data-ng-if="!$ctrl.ipBlocks.loading && !$ctrl.isAvailable()">
-            <p
-                class="oui-modal__text"
-                data-translate="vps_backup_storage_access_add_no_block"
-            ></p>
+            <p class="oui-modal__text" data-translate="vps_backup_storage_access_add_no_block"></p>
         </div>
         <div data-ng-if="!$ctrl.ipBlocks.loading && $ctrl.isAvailable()">
-            <p
-                class="oui-modal__text"
-                data-translate="vps_backup_storage_access_add_info"
-            ></p>
-            <oui-field
-                data-label="{{ ::'vps_backup_storage_access_add_ip' | translate }}"
-            >
+            <p class="oui-modal__text" data-translate="vps_backup_storage_access_add_info"></p>
+            <oui-field data-label="{{ ::'vps_backup_storage_access_add_ip' | translate }}">
                 <div class="oui-select">
-                    <select
-                        name="access-ip"
-                        id="access-ip"
-                        class="oui-select__input"
-                        multiple
-                        required
-                        data-ng-options="ip for ip in $ctrl.available track by ip"
-                        data-ng-model="$ctrl.model.ip"
-                    >
+                    <select name="access-ip" id="access-ip" class="oui-select__input" multiple required
```

Comment:
It seems like the prettier configuration was removed. 
You might want to try 
```suggestion
    prettier --write "packages/**/*.html"
```